tests/governance_js.py

```diff
@@ -1205,3 +1207,145 @@ def make_script(table_name, kind):
                     ), f"Proposal failed unexpectedly ({desc}): {msg}"
 
     return network
+
+
+@reqs.description("Test access to accepted proposal state")
+def test_final_proposal_visibility(network, args):
+    primary, _ = network.find_primary()
+    consortium = network.consortium
+
+    with temporary_constitution(
+        network,
+        args,
+        # A dumb and bad proposal that counts who voted for it. Just proving that such a thing is _possible_
+        make_action_snippet(
+            "cronyism",
```

Comment:
I would prefer if we used more neutral wording here, such as "vote_provenance". We have long had constitution samples that counted votes differently based on provenance (operator), this information is public, this is explicitly supported.